web/versioned_docs/version-0.13.0/advanced/email/email.md

```diff
@@ -224,7 +224,7 @@ Then, get the Mailgun API key and domain and add them to your `.env.server` file
 1. Go to [Mailgun](https://www.mailgun.com/) and create an account.
 2. Go to [API Keys](https://app.mailgun.com/app/account/security/api_keys) and create a new API key.
 3. Copy the API key and add it to your `.env.server` file.
-4. Go to [Domains](https://app.mailgun.com/app/domains) and create a new domain.
+4. Go to [Domains](https://app.mailgun.com/dashboard?tab=send) and create a new domain.
```

Comment:
@Reikon95 Are you sure that this URL is used to create a new domain?

I've gone to Mailgun and only found a "start page". I found the new "Add a domain" page at: https://app.mailgun.com/mg/sending/domains

Reply:
My mistake, sorry @infomiho - I think it must've temporarily had issues when I opened this issue, it's working for me too now.

Reply:
I think it still makes sense to update the link. Because the one in the current docs is incorrect: ttps://app.mailgun.com/app/domains

But you should just use the one I linked :)

Reply:
Thanks @infomiho , just added that link